components/Dialogs/src/private/MDCAlertControllerView+Private.h

```diff
@@ -31,6 +31,8 @@
 
 - (CGSize)calculatePreferredContentSizeForBounds:(CGSize)boundsSize;
 
+- (CGSize)calculateActionsSizeThatFitsWidth:(CGFloat)boundingWidth;
```

Comment:
Can you please add a comment describing what this method is doing?
Why is it needed in a private header?

Reply:
I think it is only being used in unit test.

Reply:
Then it should be named `+Testing.h` and not `+Private.h`

Reply:
I think `calculatePreferredContentSizeForBounds` also calls `calculateActionsSizeThatFitsWidth` internally, so ideally this method should be exposed in this `+Private.h` and creating another API in `+Testing.h` to call this method. I can address that but I am not sure if this is something we  usually do in our codebase.